SpiceSharp/Simulations/Simulation.cs

```diff
@@ -129,6 +129,18 @@ public enum Statuses
         // Private parameters
         private bool _cloneParameters;
 
+        /// <summary>
+        /// Gets the behavior types in the order that they are called.
+        /// </summary>
+        /// <value>
+        /// The behavior types.
+        /// </value>
+        /// <remarks>
+        /// The order is important for establishing dependencies. A behavior that is called first should
+        /// not depend on any other behaviors!
+        /// </remarks>
+        protected List<Type> BehaviorTypes { get; } = new List<Type>(10);
```

Comment:
@svenboulanger  Why `10` ?

Reply:
It is an initial guess based on the current situation (5 behavior types). It only affects performance slightly.

Maybe 10 is a little bit too much? I'll just set it to 6 or something.

Reply:
@svenboulanger I know that it's better to create a list with known size but does it matter? It's very small list.